yt/docs/README.md

```diff
@@ -1,208 +1,434 @@
-{% note info %}
+# About YTsaurus Documentation
```

Comment:
Maybe just "YTsaurus documentation"?

Reply:
I would use "About" because this section is not YTsaurus documentation itself, but it is "documentation about documentation". So this title more accurately conveys the essence of the section.